src/store/store.ts

```diff
@@ -1180,11 +1180,24 @@ export const useStore = defineStore('store', {
         const numberOfExtensionMonths = DateUtilities.subtractDates(state.stateModel.stateFilingRestoration?.expiry,
           state.stateModel.restoration?.expiry)
         const expiryDatePacific = DateUtilities.yyyyMmDdToPacificDate(state.stateModel.restoration?.expiry)
-        const formattedExpiryText = numberOfExtensionMonths + ' months, expires on ' + expiryDatePacific
-        return formattedExpiryText
+        return numberOfExtensionMonths + ' months, expires on ' + expiryDatePacific
       }
       return '[no expiry date]'
+    },
+
+    /** The court order draft file number. */
+    getCourtOrderNumberText (): string {
+      return this.stateModel.restoration.courtOrder?.fileNumber || ''
+    },
+
+    getRelationships (): RelationshipTypes[] {
+      return this.stateModel.restoration.relationships
+    },
+
+    getIsRestorationTypeCourtOrder (): boolean {
+      return !!this.stateModel.restoration.courtOrder?.fileNumber
```

Comment:
Should this be a single `!`?

Reply:
I believe a single `!` reverses a boolean value.  A double `!!` casts any type to a boolean.  Since `stateModel.restoration.courtOrder?.fileNumber` could return a string, I think a double exclamation mark is required.

Reply:
OK, I'm the one who's tired. This is well and good. Resolving.

(PS Another way to do this is `Boolean(fileNumber)`.)